pages/main.js

```diff
@@ -0,0 +1,45 @@
+document.querySelector(".login_form").addEventListener("submit", generateTable);
+
+    function generateTable(event) {
+     event.preventDefault();
+
+     console.log(12123);
+
+     const firstName = document.getElementById("first-name").value;
+     const lastName = document.getElementById("last-name").value;
+     const birthYear = document.getElementById("birth-year").value;
+     const sex = document.querySelectorAll('input[name="sex"]:checked').value;
```

Comment:
You are using `querySelectorAll` - this method returns an array of elements, then you are trying to get value in the array

Change to `querySelector`, to find one element, not an array

```
const sex = document.querySelector('input[name="sex"]:checked').value;
```